requirements-ci.txt

```diff
@@ -0,0 +1,3 @@
+pylint
+inspektor==0.5.3
```

Comment:
Actually, I also want to remove the version restriction of `inspektor`, then CI check will become real-time synchronization with upstream.